storageClient/api.go

```diff
@@ -47,6 +47,31 @@ type Group struct {
 	EncryptedGroupKey string    `json:"encrypted_group_key"`
 }
 
+// GroupAccessKeyRequest wraps the values to get or create an access key
+type GroupAccessKeyRequest struct {
+	GroupID            uuid.UUID `json:"group_id"`
+	RecordType         string    `json:"record_type"`
+	RecordID           string    `json:"record_id"`
+	EncryptedAccessKey string    `json:"encrypted_access_key"`
+	PublicKey          string    `json:"public_key"`
+}
+
+// GroupRecord wraps the values of a record for a group
+type GroupRecord struct {
+	GroupID            uuid.UUID `json:"group_id"`
+	RecordType         string    `json:"record_type"`
+	RecordID           string    `json:"record_id"`
+	EncryptedAccessKey string    `json:"encrypted_access_key"`
+	PublicKey          string    `json:"public_key"`
+}
+
+// ListGroupRecords wraps the values to list Records shared with a group
+type ListGroupRecords struct {
+	NextToken    int64
```

Comment:
ListGroupRecordsRequest or (response) so its clear when this is used without having to search for  the code that uses it.